backend/spherre/tests/test_service/test_transaction_service.py

```
@@ -0,0 +1,352 @@
+        with self.assertRaises(ValueError):
+            TransactionService.execute_transaction(
+                transaction_id=2,
+                account=self.account,
+                executor=self.member2,
+                date_executed=self.current_time + 100,
+                status=TransactionStatus.EXECUTED
+            )
+            
+    def test_reject_transaction(self):
+        # Create a test transaction
+        transaction = TransactionService.create_transaction(
```

Ruff (0.11.9)

Local variable `transaction` is assigned to but never used

Remove assignment to unused variable `transaction`

(F841)